CRM/Upgrade/Incremental/php/FiveSixtySix.php

```diff
@@ -46,6 +46,9 @@ public function upgrade_5_66_alpha1($rev): void {
     $this->addTask('Update afform tab names', 'updateAfformTabs');
     $this->addTask('Add in Client Removed Activity Type', 'addCaseClientRemovedActivity');
     $this->addTask('Update quicksearch options to v4 format', 'updateQuicksearchOptions');
+    $this->addTask('Add cache_fill_took column to Group table', 'addColumn', 'civicrm_group', 'cache_fill_took',
+      'DOUBLE DEFAULT NULL COMMENT "Seconds taken to fill smart group cache, not always related to cache_date"',
+      FALSE);
```

Comment:
@artfulrobot I believe this PR is merge-ready based on review, except that this upgrade code needs to be moved into `FiveSixtySeven.php`